courses/machine_learning/deepdive2/art_and_science_of_ml/solutions/training_models_at_scale.ipynb

```diff
@@ -27,6 +27,7 @@
       "colab": {}
```

Comment:
<p>Capitalize the "p" in Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1301/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Updated the same.